draft-ietf-jose-hpke-encrypt.md

```diff
@@ -261,20 +261,24 @@ After verification:
 # Key Encryption
 
 HPKE based recipients can be added alongside existing `ECDH-ES+A128KW` or `RSA-OAEP-384` recipients because HPKE is only used to encrypt the content encryption key, and because the protected header used in content encryption is passed to HPKE as Additional Authenticated Data.
+This protected header is formatted as BASE64URL(UTF8(JWE Protected Header)).
 
 In HPKE JWE Key Encryption:
 
-- The protected header MUST NOT contain an "alg".
-- The protected header MUST contain an "enc" that is registered in both the IANA HPKE AEAD Identifiers Registry, and the IANA JSON Web Signature and Encryption Algorithms Registry.
+- The JWE protected header MUST NOT contain an "alg". In HPKE, the algorithm used to encrypt the CEK is specified per-recipient in the recipient's header, making it unnecessary to include it in the JWE protected header.
+- The JWE Protected Header MUST include an "enc" value registered in both the IANA HPKE AEAD Identifiers Registry and the IANA JSON Web Signature and Encryption Algorithms Registry to ensure consistency across recipients (see Section 7.2.1 of {{RFC7516}}).
```

Comment:
```suggestion
- The JWE Protected Header MUST include an "enc" value registered in both the IANA HPKE AEAD Identifiers Registry and the IANA JSON Web Signature and Encryption Algorithms Registry.
```

Reply:
Its obvious that both registries are needed, you can't understand HPKE without the first, and you can't understand JWE without the second.